scripts/arango/arango-setup.sh

```diff
@@ -1,17 +1,22 @@
 #!/bin/sh
 
-if [ $1 == "test_connection" ];
+DATABASE="_system"
+USER="root"
+PASSWORD=$ARANGO_ROOT_PASSWORD
+
+if [ $1 = "test_connection" ];
+then
+  ARANGOSH_SCRIPT="check-arangosh-connection.js";
+elif [ $1 = "load_collections" ];
 then
-  ARANGOSH_SCRIPT="check-arangosh-connection.js"
-else
-  ARANGOSH_SCRIPT="setup.js"
+  ARANGOSH_SCRIPT="setup.js";
 fi
 
 # Connect to arangodb and run the arangosh setup script to load data
 arangosh \
 --server.authentication true \
---server.database _system \
---server.username root \
---server.password $ARANGO_ROOT_PASSWORD \
+--server.database $DATABASE \
+--server.username $USER \
+--server.password $PASSWORD \
 --console.history false \
---javascript.execute /home/arango-volume-share/arango/$ARANGOSH_SCRIPT
+--javascript.execute $ARANGO_DOCKER_VOLUME_SCRIPTS_DIR/$ARANGOSH_SCRIPT
```

Comment:
Provide a fallback directory. We don't want to accidentally modify our code directory.